src/dayCheck.ts

```diff
@@ -0,0 +1,11 @@
+export const FRIDAY_CELEBRATION_MESSAGE = '\n\nWe finished the week! :rocket:\nIn thread below, please answer: What are you proud of that happened this week? :muscle: Let\'s celebrate each other\'s work.';
```

Comment:
I like that this is in all caps (I know it's because it's a const), but it just makes it more exciting like we're yelling haha